server/events/apply_requirement_handler.go

```diff
@@ -0,0 +1,50 @@
+package events
+
+import (
+	"github.com/pkg/errors"
+	"github.com/runatlantis/atlantis/server/core/runtime"
+	"github.com/runatlantis/atlantis/server/events/models"
+	"github.com/runatlantis/atlantis/server/events/yaml/raw"
+	"github.com/runatlantis/atlantis/server/events/yaml/valid"
+)
+
+//go:generate pegomock generate -m --package mocks -o mocks/mock_apply_handler.go IAggregateApplyRequirements
+type IAggregateApplyRequirements interface {
```

Comment:
I would just call this `ApplyRequirement`. AggregateApplyRequirements is just one implementation of an ApplyRequirement where it just handles all the apply requirements.  But there could be other implementations of that as well.

Reply:
Done!